### PR TITLE
NNEO-10174 VendoSdk - Implemented the Reconciliation API client

### PR DESCRIPTION
**Pull request by adolfoabegg**

### CODE REVIEW

File: examples/reporting/get_transaction_details.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        if (!empty($row)) {
+            //Do something with the data
+            echo "Vendo Transaction ID = " . $row->transaction['id'] . ", ";
+            echo "Merchant reference = " . $row->transaction->merchantReference . ", ";
+            echo "Vendo Subscription ID = " . $row->subscription['id'] . ", ";
+            echo "Subscription Username = " . $row->subscription->username . "\n";
+            echo "Vendo Customer ID = " . $row->customer['id'] . ", ";
+            echo "Customer First name = " . $row->customer->firstname . ", ";
+            echo "Customer Email address = " . $row->customer->email . ", ";
+            if (in_array($row->transaction->type , [30, 31])) {//is it a refund or a partial refund?
```
Comment (zbigniewwiech) [inline examples/reporting/get_transaction_details.php:25]: pls describe meaning of 30,31 flags (30 full refund, ...)

File: examples/reporting/get_transaction_details.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                echo "Refund reason = " . $row->transaction->refundReason . ", ";
+            }
+            if ($row->transaction->type == 20) {//is it a chargeback?
```
Comment (zbigniewwiech) [inline examples/reporting/get_transaction_details.php:28]: ditto

File: examples/reporting/get_transaction_details.php (hunk 1 above)
Comment (adolfoabegg) [reply examples/reporting/get_transaction_details.php]: I've added constants for the transaction types

File: examples/reporting/get_transaction_details.php (hunk 2 above)
Comment (adolfoabegg) [reply examples/reporting/get_transaction_details.php]: used the constants here too..

**Review by zbigniewwiech — APPROVED** [review]

**Pull request merged**